src/glossary.md

```diff
@@ -478,7 +479,8 @@ endpoints:
 - [Private APIs](#private-api) allow configuring the service locally.
 
 External entities such as [light clients](#light-client) can access endpoints
-via REST API. The configuration for REST API is specified in the service.
+via REST API and WebSockets. The configuration for REST API and WebSockets is
+specified in the service.
 
```

Comment:
There are two categories of endpoints. 1. Pre-defined on core level. E.g. (`/explorer/v1/transactions`, `system/v1/health etc. BY THE WAY WebSocket at this level as well) 2. Custom endpoints which configure on a service level by developer of the service. E.g. (`/service/cryptocurrency/v1/wallets`).